webview/src/experiments/components/table/styles.module.scss

```diff
@@ -626,57 +629,21 @@ $badge-size: 0.85rem;
   font-size: 0.65rem;
 }
 
-.bullet {
-  visibility: visible;
-  align-items: center;
-  background: $checkbox-background;
-  border-radius: $bullet-size;
-  border: calc(var(--border-width) * 1px) solid $checkbox-border;
-  color: $icon-color;
+.plotBox {
```

Comment:
A couple comments on the design:

<img width="175" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/3e18d716-791f-48c7-9b8a-cfd91a4e026f">